integration-tests/Makefile

```diff
@@ -68,7 +68,7 @@ build_test_image:
 #Build a chainlink docker image for local testing and push to k3d registry
 .PHONY: build_push_docker_image
 build_push_docker_image:
-	docker build -f ../core/chainlink.Dockerfile --build-arg COMMIT_SHA=$(git rev-parse HEAD) --build-arg CHAINLINK_USER=chainlink -t localhost:5000/chainlink:develop ../ ; docker push localhost:5000/chainlink:develop
+	docker build -f ../core/chainlink.Dockerfile --build-arg COMMIT_SHA=$(git rev-parse HEAD) --build-arg CHAINLINK_USER=chainlink -t chainlink:develop ../
```

Comment:
Seems we have another command here that does the same - https://github.com/smartcontractkit/chainlink/pull/10550/files#diff-78c82fbb6537a0de6be7bedb6d39e3930befe3750dc8fd7ba15cc7b23d12ea2dR197-R200
We can use that command. This change is not needed. Maybe this command can be removed, since local testing on k8s is not used anymore.